src/main/metabase/examples.ts

```diff
@@ -191,8 +191,54 @@ function columnExampleQueries(field: Field, table: Table, aidColumns: string[]):
   }
 }
 
+function interactionSQL(fieldA: Field, fieldB: Field, table: Table): ExampleQuery {
```

Comment:
`interactionSQL` is not a descriptive name.

Reply:
> pivotTableSQL? I was out of ideas tbh

Maybe dumb and simple: `groupBy2Columns`?

If both columns have few values could we make a fancy chart like this:

![image](https://user-images.githubusercontent.com/12145268/205663765-f8375a65-5ea5-4e2e-98b3-a69b8b6a0c8c.png)

Reply:
Code for above chart:
```
display: 'bar',
visualization_settings: {
  "graph.dimensions": [
    "school",
    "gender"
  ],
  "graph.metrics": [
    "count"
  ]
}
```

Reply:
nice, I was looking for this but didn't dig deep enough. I'll use this and I think lower distinct should be the color-coded one.

I also got a late thought that we should have an upper limit in the big `.filter` clause, so that we don't show columns with > 20 (?) distinct values like that

Reply:
If both axes have low distinct count, we show the chart, else we show a table. In both cases the horizontal axis should be small.